tests/API/Management/JobsTest.php

```diff
@@ -135,7 +135,7 @@ public function testThatSendVerificationEmailIsFormedProperly()
     {
         $api = new MockManagementApi( [ new Response( 200, self::$headers ) ] );
 
-        $api->call()->jobs()->sendVerificationEmail( '__test_user_id__' );
+        $api->call()->jobs()->sendVerificationEmail( '__test_user_id__', [ 'client_id' => '__test_client_id__' ] );
```

Comment:
Shouldn't it be better to have a test without `client_id` and another with? Not a blocker.

Reply:
Would be a bit more complete, yes, but this does cover both cases. I'm not sure I'd want to add a separate test for all the different combinations of parameters possible.